<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lacewing_dialog</class>
 <widget class="QMainWindow" name="Lacewing_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lacewing @ Imperial College London</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="system_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QGroupBox" name="system_groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>System Settings</string>
     </property>
     <widget class="QLabel" name="serial_port_title_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Serial Port:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="serial_port_comboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <item>
       <property name="text">
        <string>Refresh</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="ui_status_title_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="ui_status_label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>IDLE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="console_frame">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QGroupBox" name="console_groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>401</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Console</string>
     </property>
     <widget class="QPushButton" name="console_clear_pushButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>230</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>69</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>69</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="console_textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>201</height>
       </rect>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="user_cmd_groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Debug Interface for Designer</string>
     </property>
     <widget class="QLineEdit" name="user_cmd_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string comment="Only used for designer"/>
      </property>
      <property name="text">
       <string>Input your command here</string>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="user_cmd_pushButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>69</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>69</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string>send</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>user_cmd_lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>user_cmd_pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>643</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
